src/UKHO.ERPFacade.Common/Helpers/AzureBlobEventWriter.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Azure.Storage.Blobs;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json.Linq;
+using System.Diagnostics.CodeAnalysis;
+using System.Text;
+using UKHO.ERPFacade.Common.Configuration;
+using UKHO.ERPFacade.Common.Logging;
+
+namespace UKHO.ERPFacade.Common.Helpers
```

Comment:
Could we change the namespace to UKHO.ERPFacade.Common.IO or something similar? Thanks!

Reply:
Changed the namespace for all four classes to UKHO.ERPFacade.Common.IO